### PR TITLE
mgr/nfs: annotate _cmd_nfs_* methods return value 

### PR DESCRIPTION
**Pull request by varshar16**
Fixes: https://tracker.ceph.com/issues/50783
Signed-off-by: Varsha Rao <varao@redhat.com>

~Tests need to be updated.~

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by batrick — CHANGES_REQUESTED** [review]
Please note `Fixes: 7c57d1634049484b3037d297e0955b14dcac9654` in `mgr/nfs: annotate _cmd_nfs_* methods return value`.

**Review by liewegas — APPROVED** [review]
changes look good!

File: src/pybind/mgr/nfs/module.py [hunk 1]
```diff
@@ -24,65 +25,65 @@ def __init__(self, *args, **kwargs):
 
     @CLICommand('nfs export create cephfs', perm='rw')
     def _cmd_nfs_export_create_cephfs(self, fsname: str, clusterid: str, binding: str,
-                                      readonly: bool=False, path: str='/'):
+                                      readonly: bool=False, path: str='/') -> Tuple[int, str, str]:
```
Comment (tchaikov) [inline src/pybind/mgr/nfs/module.py]: could you take this opportunity to annotate all the functions in this source file and add an section in `src/mypy.ini` to ensure that this file is annotated in future?

Comment (varshar16) [reply src/pybind/mgr/nfs/module.py]: I will do it in a different PR. I have created a tracker ticket for it: https://tracker.ceph.com/issues/50816

**Comment by varshar16** [comment]
> Please note `Fixes: 7c57d1634049484b3037d297e0955b14dcac9654` in `mgr/nfs: annotate _cmd_nfs_* methods return value`.

Done.

**Review by batrick — APPROVED** [review]

**Comment by varshar16** [comment]
Tests Passed: https://pulpito.ceph.com/varsha-2021-05-17_04:22:27-rados-wip-annotate-ret-vals-nfs-distro-basic-smithi/

**Pull request merged**